core/templates/css/oppia.css

```diff
@@ -1390,8 +1390,8 @@ span.sort-explorations-select .sort-order {
 .oppia-save-draft-button.btn-success,
 .oppia-save-changes-button.btn-success,
 .oppia-editor-publish-button.btn-success {
-  background-color: #28a745;
-  border-color: #28a745;
+  background-color: #24883e;
+  border-color: #24883e;
```

Comment:
I don't think we should just introduce new colors to our liking to solve accessibility. it will snowball into a bunch of corner cases which will later make developer experience worse. Also, this might be rather painful to track and clear off later. @seanlip I see your comment approving this change here https://github.com/oppia/oppia/issues/17886#issuecomment-1493304296. Can we get some guidance from the design team? Asking this because this color is also used in `.toast-success`. Different (but close enough) colors might make the page look inconsistent.

Reply:
@srijanreddy98 Good points -- I think what you say makes sense for the learner-facing pages and the contributor dashboard, but I'm less concerned about it for the editor view because that hasn't been subject to a full design treatment yet, so I think the solution here is still probably OK (since we still should resolve the accessibility issues in the meantime). If the colour is already used elsewhere then that actually makes things better.